tarzst.go

```diff
@@ -22,6 +25,16 @@ func (*TarZstd) CheckExt(filename string) error {
 	return nil
 }
 
+// SetEncoderOptions allows you to configure the underlying zstandard encoder
+func (t *TarZstd) SetEncoderOptions(opts ...zstd.EOption) {
+	t.eopts = opts
+}
```

Comment:
Why not just export the fields in the first place like all the existing config fields?

Reply:
Right! Hadn't seen the other ones, even better.  What about `EncoderOptions []zstd.EOption` and `DecoderOptions` ?

Reply:
Sounds good